kubernetes/resource_kubernetes_csi_driver.go

```diff
@@ -0,0 +1,280 @@
+package kubernetes
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	api "k8s.io/api/storage/v1beta1"
```

Comment:
It's inconsistent across resources at the moment, but I would explicitly call this `storage` rather than just `api`.

Reply:
done